ci/ci_cli.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import subprocess
+
+# import sdk_install
+import logging
+
+from concurrent import futures
+
+MAX_WORKERS = 10
+
+JOB_TYPE_INSTALL = 'install'
+JOB_TYPE_DIAGNOSTIC = 'diagnostic'
+
+JOB_TYPES = [JOB_TYPE_INSTALL, JOB_TYPE_DIAGNOSTIC]
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def install(service_name):
+    import sdk_install
+
+    log.info("Starting %s installation.", service_name)
+
+    sdk_install.install(
+        service_name,
+        service_name,
+        0,
+        package_version=sdk_install.PackageVersion.LATEST_UNIVERSE
+    )
+
+
+def diagnostic(service_name):
+    log.info("Starting %s diagnostic.", service_name)
+
+    src = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        '..', 'python', 'create_service_diagnostics_bundle.sh'
+    )
+
+    cmd = '{src} --package-name={service} --service-name={service} --yes'.format(
+        src=src,
+        service=service_name)
+
+    process = subprocess.Popen(
+        [cmd],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        shell=True,
+        universal_newlines=True)
+
+    # print execution output
+    while True:
+        output = process.stdout.readline()
```

Comment:
You're not reading stderr, which could lead to a deadlock.
In fact, I don't think this adds any value, so why not call `Popen` with default `std{err,out}` such that they are printed directly?

Reply:
yes, you are right, changed